metaflow/plugins/aws/step_functions/schedule_decorator.py

```diff
@@ -9,13 +9,13 @@ class ScheduleDecorator(FlowDecorator):
 
     Parameters
     ----------
-    hourly : bool
-        Run the workflow hourly (default: False).
-    daily : bool
-        Run the workflow daily (default: True).
-    weekly : bool
-        Run the workflow weekly (default: False).
-    cron : str
+    hourly : bool, default: False
+        Run the workflow hourly.
+    daily : bool, default: True
+        Run the workflow daily.
+    weekly : bool, default: False
+        Run the workflow weekly.
+    cron : str, optional
         Run the workflow at [a custom Cron schedule](https://docs.aws.amazon.com/eventbridge/latest/userguide/scheduled-events.html#cron-expressions)
         specified by this expression.
     timezone : str
```

Comment:
timezone is also optional

Reply:
ah right, it was added after this PRR was done. I'll add it.